rcl/src/rcl/node.c

```diff
@@ -542,6 +554,147 @@ rcl_get_disable_loaned_message(bool * disable_loaned_message)
   *disable_loaned_message = (strcmp(env_val, "1") == 0);
   return RCL_RET_OK;
 }
+
+void rcl_node_type_description_service_handle_request(
+  rcl_node_t * node,
+  const rmw_request_id_t * request_header,
+  const type_description_interfaces__srv__GetTypeDescription_Request * request,
+  type_description_interfaces__srv__GetTypeDescription_Response * response)
+{
+  (void)request_header;
```

Comment:
Nit; I don't think we need this, since we are using the type in the `RCL_CHECK_FOR_NULL_WITH_MSG` call below.